web/post-restoration-transition-application/nuxt.config.ts

```diff
@@ -115,10 +113,6 @@ export default defineNuxtConfig({
       version: `Post Restoration Transition Application UI v${process.env.npm_package_version || ''}`,
       preexistingCompanyProvisions: process.env.PREEXISTING_COMPANY_PROVISIONS_URL
```

Comment:
Can you remove this and add it to the env + devops file? I added it to 1pass so you can reference it now: 
```
NUXT_PUBLIC_PREEXISTING_COMPANY_PROVISIONS_URL=NUXT_REGISTRY_HOME_URL="op://web-url/$APP_ENV/bcregistry/PREEXISTING_COMPANY_PROVISIONS_URL"
```